gbn/gbn_server.go

```diff
@@ -39,7 +39,9 @@ func NewServerConn(ctx context.Context, sendFunc sendBytesFunc,
 // 2.  The server then responds with a SYN message.
 // 3.  The server waits for a SYNACK message from the client.
 // 4a. If the server receives the SYNACK message before a resendTimeout, the hand
-//     is considered complete.
+//
+//	is considered complete.
+//
```

Comment:
Unintended reformatting of the docs?

Reply:
yeah 🙈  (edited it now to make my formatter happy)